src/front/wgsl/mod.rs

```diff
@@ -439,6 +452,14 @@ impl<'a> Error<'a> {
                 labels: vec![(span.clone(), "expression is not a reference".into())],
                 notes: vec![],
             },
+            Error::InvalidAssignment{ ref span, ty} => ParseError {
+                message: "invalid left-hand side of assignment".into(),
+                labels: vec![(span.clone(), "cannot assign to this expression".into())],
+                notes: match ty {
+                    InvalidAssignmentType::Swizzle => vec!["WGSL does not support assignments to swizzles".into(), "trying assigning each component individually".into()],
```

Comment:
"try assigning"

Reply:
I changed this to "consider assigning", would you prefer it to be "try assigning"?

Reply:
Nope, just wanted the "trying assigning" typo fixed. "consider" is fine too.